packages/clerk-js/rspack.config.js

```diff
@@ -65,7 +68,7 @@ const common = ({ mode, disableRHC = false }) => {
         }),
     ].filter(Boolean),
     output: {
-      chunkFilename: `[name]_[fullhash:6]_${packageJSON.version}.js`,
+      chunkFilename: `[name]_${variant}_[fullhash:6]_${packageJSON.version}.js`,
```

Comment:
This is so that we can distinguish which variant a chunk belongs to, mainly so we could (in the future) target specific variant chunks with specific bundlewatch rules.